dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::V1::SectionsController < ApplicationController
+  layout false
+
+  # GET /api/v1/sections
+  # Get the set of sections owned by the current user
+  def index
+    return head :forbidden unless current_user
+    render json: current_user.sections.map(&:summarize)
+  end
+
+  # GET /api/v1/sections/<id>
+  # Get complete details of a particular section
+  def show
+    if !current_user || !current_user.teacher?
+      head :forbidden
+    else
```

Comment:
:+1: You're right, and it looks like in a later branch I update this.  I'll pull that change into this PR.